crypto/dilithium/p_dilithium3.c

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
another general comments: it'd be nice to add a short example how to use dilithium to the readme file (similarly to https://github.com/awslabs/aws-lc/blob/main/crypto/kyber/README.md).

Reply:
Thanks, I've followed the structure of your README and updated in https://github.com/awslabs/aws-lc/pull/637/commits/5d44b6ed8f3a495d1bca9daa293bace87e1c8871.